3.Tests/UI/Features/Item/Update.feature

```diff
@@ -4,9 +4,9 @@ Feature: Item Update
     Background:
         Given the user is logged in
 
-    @Smoke @Regression @create.project.NewProject @create.item
+    @Smoke @Regression @create.project.Exercise @create.item @update
```

Comment:
Is the last tag @update supposed to be the id of this test? if so it should be more descriptive, like @UI_Update_Item

Reply:
Done, thank you